### PR TITLE
Update README with new Vercel Deploy Buttons

### PR DESCRIPTION
**Pull request by jakelumetta**

### CODE REVIEW

**Comment by orlyohreally** [comment]
@jakelumetta, should I review it after https://github.com/ButterCMS/buttercms/pull/1269 is merged?

**Comment by jakelumetta** [comment]
@orlyohreally Yes - once that code is on prod you can test the button

File: README.md [hunk 1]
```diff
@@ -13,76 +13,33 @@ Once created, this project can be easily and quickly deployed to Vercel from the
...
 
-```bash
-npm run dev # or yarn dev
-```
-
-To quit the server press CONTROL-C.
-
-### Build the app
-Use one of these commands which build the application for production usage:
+To view the app in the browser, you'll need to run the local development server:
```
Comment (ViolanteCodes) [inline README.md]:
```suggestion
To view the app in a browser, you'll need to run the local development server:
```
If you're not referring to a specific browser, I believe the indefinite article is likely correct.

File: README.md [hunk 2]
```diff
@@ -13,76 +13,33 @@ Once created, this project can be easily and quickly deployed to Vercel from the
...
-## Webhooks to trigger deployment
 
-Use of the ButterCMS webhooks to notify the application every time a post, page or collection item are added or modified, and reload the Next.js application.
+Congratulations! Your starter project is now live: [http://localhost:3000](http://localhost:3000).
 
-You can add a new webhook by going to the [https://buttercms.com/webhooks/](https://buttercms.com/webhooks/) page.
+## 4) Deploy
 
-![webhook page](https://cdn.buttercms.com/ixt8yfOsSgWLlwc0eis1)
+Deploy your Butterized proof of concept app and to spread your love of Butter.
```
Comment (ViolanteCodes) [inline README.md]:
```suggestion
Deploy your Butterized proof of concept app and spread your love of Butter.
```
Either "and" or "to"

File: README.md [hunk 3]
```diff
@@ -13,76 +13,33 @@ Once created, this project can be easily and quickly deployed to Vercel from the
...
 
-If you use Vercel, you can create deploy webhooks there. ![Vercel deploy webhooks](https://cdn.buttercms.com/bPIzuIXTleK7h4Kx3X4L)
+###  Vercel
 
-Learn more about webhooks [here](https://buttercms.com/docs/api/#webhooks).
+Deploy your Next.js app using Vercel the creators of Next.js. This process will create a copy of the this starter project in your Git provider account, instantly deploy it, and have a full content workflow connected to your ButterCMS account. Smooth.
```
Comment (ViolanteCodes) [inline README.md]:
```suggestion
Deploy your Next.js app using Vercel, the creators of Next.js. With the click of a button, you'll create a copy of your starter project in your Git provider account, instantly deploy it, and institute a full content workflow connected to your ButterCMS account. Smooth.
```

These are little nitpicky things - obviously the wording is up for grabs, but there were some places where the subject/verb agreement weren't super clear, so I just went in and tweaked those.

**Review by ViolanteCodes — COMMENTED** [review]
Just some typos and suggestions on wording?

**Review by orlyohreally — APPROVED** [review]
The button worked for me https://share.getcloudapp.com/5zunWEw2, very nice flow!

**Pull request merged**